### PR TITLE
Feature: Add environment in contract id

### PR DESCRIPTION
**Pull request by fedeesti**
### Summary

- Add receive an environment contract ID and convert it to a value

### Details

- Add function to transform `environment` in `contractId` 
- Add find one environment by name

### CODE REVIEW

File: src/modules/invocation/domain/invocation.domain.ts [hunk 1]
```diff
@@ -43,4 +43,10 @@ export class Invocation extends Base {
     this.id = id;
     this.selectedMethod = selectedMethod;
   }
+
+  getContractIdValue = (inputString: string): string => {
+    const regex = /{{(.*?)}}/g;
+    const contractId = inputString.replace(regex, (match, text) => text);
+    return contractId;
+  };
 }
```
Comment (leotammaro) [inline src/modules/invocation/domain/invocation.domain.ts:52]: You must to create unit test to this method

File: src/modules/invocation/domain/invocation.domain.ts [hunk 2]
```diff
@@ -43,4 +43,10 @@ export class Invocation extends Base {
     this.id = id;
     this.selectedMethod = selectedMethod;
   }
+
+  getContractIdValue = (inputString: string): string => {
+    const regex = /{{(.*?)}}/g;
```
Comment (leotammaro) [inline src/modules/invocation/domain/invocation.domain.ts:48]: add `//comment` what this regex do

Comment (fedeesti) [reply src/modules/invocation/domain/invocation.domain.ts:48]: The test I created in invocation.e2e.spec.ts demonstrates this functionality

File: src/modules/invocation/domain/invocation.domain.ts (hunk 1 above)
Comment (fedeesti) [reply src/modules/invocation/domain/invocation.domain.ts:52]: The test I created in invocation.e2e.spec.ts demonstrates this functionality

**Review by leotammaro — APPROVED** [review]

**Pull request merged**